### PR TITLE
WSO-525 - autogenerate path for page builder

### PR DESCRIPTION
**Pull request by dpfitzsi**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_page_builder/ucgy_core_page_builder.module [hunk 1]
```diff
@@ -9,6 +9,29 @@
...
+  $items['page-builder/path'] = array(
+    'page callback' => 'ucgy_core_page_builder_menu_page_callback',
+    'type' => MENU_CALLBACK,
+    'access arguments' => array('access content'),
+  );
+  return $items;
+}
+
+/* this method is defined in ucgy_core_page_builder_menu()
+ * require_once - includes a reference to the pathauto module so we can call the pathauto_cleanstring()
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_page_builder/ucgy_core_page_builder.module]: Remove the "the" in "call the pathauto_cleanstring()"

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_page_builder/ucgy_core_page_builder.module [hunk 2]
```diff
@@ -9,6 +9,29 @@
...
+ * pathauto_cleanstring() - uses the constant var of all the small words to be removed
+ * */
+function ucgy_core_page_builder_menu_page_callback() {
+  require_once drupal_get_path('module', 'pathauto') . '/pathauto.inc';
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_page_builder/ucgy_core_page_builder.module]: Should use module_load_include instead https://api.drupal.org/api/drupal/includes%21module.inc/function/module_load_include/7.x

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_page_builder/ucgy_core_page_builder.module (hunk 1 above)
Comment (SandyVallee) [reply sites/all/modules/ucalgary/ucgy_core/ucgy_core_page_builder/ucgy_core_page_builder.module]: ok removed from comment

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_page_builder/ucgy_core_page_builder.module (hunk 2 above)
Comment (SandyVallee) [reply sites/all/modules/ucalgary/ucgy_core/ucgy_core_page_builder/ucgy_core_page_builder.module]: having problems but maybe I'm not formatting pararms correctly? 
module_load_include('module', 'pathauto', 'pathauto.module');

Comment (SandyVallee) [reply sites/all/modules/ucalgary/ucgy_core/ucgy_core_page_builder/ucgy_core_page_builder.module]: done

**Pull request merged**